.github/workflows/progress-tracker.yml

```diff
@@ -0,0 +1,61 @@
+name: Progress Tracker
+
+on:
+  workflow_dispatch:
+  issues:
+    types:
+      - edited
+
+jobs:
+  track-progress:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v2
+        with:
+          script: |
+            // Update the following two constants each year
+            const FILTER_LABEL = '2020 chapter'
+            const TRACKER_ISSUE_ID = 989
+
+            const parseTasks = text => {
+              const tasks = []
+              text.split('\n').forEach(line => {
+                if (/^(\s{2,})?([*+-]|\d+\.)\s+\[ \]\s+\S+/.test(line)) {
+                  tasks.push('☐')
+                }
+                if (/^(\s{2,})?([*+-]|\d+\.)\s+\[[xX]\]\s+\S+/.test(line)) {
+                  tasks.push('☑')
+                }
+              })
+              return tasks
+            }
+
+            const taskIssues = await github.paginate(github.issues.listForRepo, {
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              direction: 'asc',
+              labels: FILTER_LABEL
+            })
+
+            let report = ''
+            for (const issue of taskIssues) {
+              tasksStatus = parseTasks(issue.body)
+              if(!report) {
+                report = `Milestones | ${tasksStatus.map((v, i) => i + 1).join(' | ')}\n`
```

Comment:
Could we also parse the draft doc URL? For some issues it's still `#`.

If the doc URL is defined, it should appear as a clickable link, like "[Doc]()". But if the  URL is `#` it should just be blank, with no "Doc" word.

Reply:
I have actually added code to parse other URLs as well, in case we need them in the future.

Rather than consuming yet another grid column and repeat `Doc` in each row, I added draft link with an icon next to the chapter title like:

[JavaScript](#) • [🗎](#)

Reply:
I have further updated it to list it in a column as I have also added a column for teams.